games-roguelike/FTL-gog/FTL-gog-1.6.7.ebuild

```diff
@@ -7,7 +7,7 @@ inherit desktop unpacker
 
 DESCRIPTION="Top-down roguelike space ship simulator"
 HOMEPAGE="https://www.gog.com/game/faster_than_light"
-SRC_URI="ftl_advanced_edition_en_1_6_3_17917.sh"
+SRC_URI="ftl_advanced_edition_en_1_6_7_18662.sh"
```

Comment:
Could you use PV to generate this? Possible add the last bit to PV, and do e.g. `${PV//./_}`?

Reply:
Yep, works quite nicely, made the switch.